guides/v2.0/extension-dev-guide/code-generation.md

```diff
@@ -26,15 +26,11 @@ Provided the Magento application is not set for <a href="{{ site.gdeurl }}config
 
 In particular,
 
-*	A <a href="{{ site.gdeurl }}extension-dev-guide/depend-inj.html#dep-inj-mod-type-fact">Factory</a> class creates instances of a type. For example, a generated <code>\Magento\Customer\Model\AddressFactory</code> creates new instances of <code>\Magento\Customer\Model\Address</code>. The code in <code>AddressFactory</code> provides "typeSafety" (@return annotation) so IDEs understand the type of returned object for the <code>Address</code> type.
+*	A Factory class creates instances of a type. See <a href="{{ site.gdeurl }}extension-dev-guide/factories.html">Instantiating objects with factories</a> for more information. Factories are directly referenced within application code.
 
+*	You can designate a Proxy to be generated for a type in order to ensure the type is not instantiated until it is needed. See <a href="{{ site.gdeurl }}extension-dev-guide/proxies.html">Proxies</a> for more information. Proxies are directly referenced within application code.
```

Comment:
@Andon

Good catch.  By that statement I intended to distinguish it from generated
classes you do not reference at all, but "application code" was incorrect.
I fixed the phrasing in the PR branch.


Chris Nanninga
Senior Engineer, Classy Llama
MAGENTO CDP, Magento CFED, Magento CSS, Zend Ce
*o* 417.866.8887 ext. 105 | *chris.nanninga@classyllama.com
<chris.nanninga@classyllama.com>*

On Sun, Feb 21, 2016 at 11:12 AM, Anton Kril <notifications@github.com>
wrote:

> In guides/v2.0/extension-dev-guide/code-generation.md
> <https://github.com/magento/devdocs/pull/535#discussion_r53569557>:
>
> >
> > +*	You can designate a Proxy to be generated for a type in order to ensure the type is not instantiated until it is needed. See <a href="{{ site.gdeurl }}extension-dev-guide/proxies.html">Proxies</a> for more information. Proxies are directly referenced within application code.
>
> Proxies should never be directly referenced in code. Only in configuration
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/magento/devdocs/pull/535/files#r53569557>.
>